src/java/org/apache/cassandra/utils/memory/MemtableCleanerThread.java

```diff
@@ -84,4 +118,11 @@ public void trigger()
     {
         trigger.run();
     }
+
+    /** Return the number of pending tasks */
+    @VisibleForTesting
+    public int numPendingTasks()
+    {
+        return this.clean.numPendingTasks();
```

Comment:
Nit: `this` is not needed:
```suggestion
        return clean.numPendingTasks();
```